src/DevBetterWeb.Infrastructure/Handlers/DailyCheckInitiatedEventHandler.cs

```diff
@@ -46,12 +53,31 @@ public async Task Handle(DailyCheckInitiatedEvent domainEvent)
     // check if number of MemberSubscriptionPlans == expected number
     await _dailyCheckSubscriptionPlanCountService.WarnIfNumberOfMemberSubscriptionPlansDifferentThanExpected(messages);
 
+    await UpdateVideosThumbnail();
```

Comment:
If this thing fails I definitely *do not* want all the other things that happen in this process to not happen.

Reply:
added try catch inside UpdateVideosThumbnail